cni/pkg/nodeagent/net_test.go

```diff
@@ -156,7 +156,7 @@ func TestServerAddPod(t *testing.T) {
 		netip.MustParseAddr("99.9.9.9"),
 		uint8(unix.IPPROTO_TCP),
 		string(podMeta.UID),
-		true,
+		false,
```

Comment:
Why do we flip the boolean value for all of these tests?

Reply:
It's the expected call mock (so we don't have to actually keep track of  a real `ipset` as unit test state machinery) - we flipped it in the code, so we have to flip it for all the `expect`s